src/app/users/update-business-map-info.ts

```diff
@@ -0,0 +1,20 @@
+import { AccountsRepository } from "@services/mongoose/accounts"
+
+export const updateBusinessMapInfo = async ({
+  username,
+  coordinate,
+  title,
+}: {
+  username: string
+  coordinate: { latitude: number; longitude: number }
+  title: string
+}): Promise<Account | Error> => {
+  const accountsRepo = AccountsRepository()
+
+  const account = await accountsRepo.findByUsername(username as Username)
+  if (account instanceof Error) return account
+
+  account.coordinate = coordinate as Coordinates
+  account.title = title as BusinessMapTitle
+  return accountsRepo.update(account)
```

Comment:
this kind of update is not good at account level... please use a specific repo for businesses (this will facilitate the subsequent migration)

Reply:
is that not good enough for now? we want to migrate all the use cases to app shortly, then we can improve those.